### ISSUE TITLE
Cannot call DaxStudio 2.7 from vba

### ISSUE THREAD

**Issue by bdarbonneau** [DaxStudio/DaxStudio#14]
Darren,
I am sorry I missed the beta testing period. It seems that there was a minor regression in the v2.7: it used to be possible to launch the Excel addin with the following vba code, but it now (v2.7) raises an error: 
```
Dim addIn As COMAddIn
Dim automationObject As Object

Set addIn = Application.COMAddIns("DaxStudio.ExcelAddIn")
If Not addIn Is Nothing Then
    Set automationObject = addIn.Object
    automationObject.Launch
End If
```
Ref: [this old thread on Dax Studio codeplex site](http://daxstudio.codeplex.com/discussions/644003)

Regards,
Bertrand

**Comment by dgosbell** [comment]
hmm, yes I added an extra parameter to the launch method to allow for the enabling of diagnostic logging. I've added back the default launch method which just sets this to false by default in v2.7.1 - I have a build of this up on my [OneDrive](https://1drv.ms/f/s!AnSKp2UbblSYgZsQ_xzxrPM-A1gfmQ) if you'd like to test it?

**Comment by dgosbell** [comment]
This should now be fixed in the 2.7.1 release
